src/sass/_common/common.scss

```diff
@@ -638,6 +638,21 @@ label + input {
     }
 }
 
+#signup_form {
```

Comment:
does it belong under common? seems only specific to this page, might be more suited for pages.scss, or websocket_pages.scss depending